caffe2/core/nomnigraph/Converters/Caffe2.cc

```diff
@@ -13,44 +13,54 @@ getArgumentsFromOperator(caffe2::OperatorDef op) {
   return argMap;
 }
 
+nom::repr::NeuralNetOperator::NNLayout getLayout(std::map<std::string, caffe2::Argument> argMap) {
+  if (argMap.count("order")) {
+    auto order = argMap["order"].s();
+    if (order == "NCHW") {
```

Comment:
You should also check for "nchw" string, or use https://github.com/pytorch/pytorch/blob/a2a28c0ef1d9a433972fe72fa5b0b9b850ccfcaf/caffe2/core/types.h#L22-L31

Reply:
ah, didn't realize that was a valid layout string